lib/trento_web/router.ex

```diff
@@ -88,6 +88,11 @@ defmodule TrentoWeb.Router do
   scope "/api", TrentoWeb.V1 do
     pipe_through [:api, :api_v1]
     get "/public_keys", SettingsController, :get_public_keys
+
+    # Analytics Settings
```

Comment:
Duplicated. Let's remove this addition

Reply:
Why do we even want that? Either way, we need to change it. We must not have the same endpoint repeated twice. We will need to think how to put it but duplicating is a no-go